daemon/container_operations.go

```diff
@@ -246,12 +246,10 @@ func (daemon *Daemon) updateNetworkSettings(ctr *container.Container, n *libnetw
 			// Avoid duplicate config
 			return nil
 		}
-		if !containertypes.NetworkMode(sn.Type()).IsPrivate() ||
-			!containertypes.NetworkMode(n.Type()).IsPrivate() {
```

Comment:
Is this causing the linter to complain? To be honest I find the previous one to be more readable.

Reply:
This one not, but it made my brain complain as I initially thought there was some nested conditions here, and having them on a single line made that more clear to me.

Do you want me to change it back? I can drop this commit if you want to.

Reply:
It makes it much more visible that it's the same check, just on the different variable `sn` vs `n`.
I would be more happy to drop it, but it's not a blocker also, so I'll leave it up to you.

Reply:
I dropped the commit 👍